app/controllers/health/patient_controller.rb

```diff
@@ -22,8 +22,8 @@ def index
 
     def update
       @patient.update(patient_params)
-      @patient.build_team_member!(Health::Team::CareCoordinator, patient_params[:care_coordinator_id], current_user) if patient_params[:care_coordinator_id].present?
-      @patient.build_team_member!(Health::Team::Nurse, patient_params[:nurse_care_manager_id], current_user) if patient_params[:nurse_care_manager_id].present?
+      @patient.build_team_member!(Health::Team::CareCoordinator, patient_params[:care_coordinator_id].to_i, current_user) if patient_params[:care_coordinator_id].present?
```

Comment:
Throwing `to_i` on here will turn `nil` into `0` is that intentional?

Reply:
Team members should only be built if there is one present.